document/CoreFormat.mdk

```diff
@@ -413,6 +413,44 @@ The `results` array shall be empty if the run that produced the `run` object did
 If the tool was run solely for the purpose of exporting rule metadata (see &sect;[#run-rules]),
 the `results` property shall be absent.
 
+### `toolNotifications` { #run-toolNotifications }
+
+A `run` object may contain a property named `toolNotifications`
+whose value is an array of zero or more `notification` objects (&sect;[#notification]).
```

Comment:
No, you're right, I completely failed to point out the whole "audience" thing. Will fix.